nengo_interfaces/tests/test_changing_airsim_settings.py

```diff
@@ -0,0 +1,85 @@
+# NOTE tests if the airsim interface warns the user if a setting.json parameter changes and requires
+# a UE4 sim restart
+# make sure that the Capture Settings: Width parameter does not match what you current have in your
+# ~/Documents/Airsim/settings.json
+import json
+import math
+from os.path import expanduser
+
+import nengo
+import numpy as np
+
+from nengo_interfaces.airsim import AirSim
+
+# read the current capture width so we can:
+# - change it to test if we get warned, change it back
+# - test that we do not get an exception if it doesn't change
+home = expanduser("~")
+with open(  # pylint: disable=W1514
+    f"{home}/Documents/AirSim/settings.json", "r+"
+) as fp:
+    data = json.load(fp)
+    prev_width = data["CameraDefaults"]["CaptureSettings"][0]["Width"]
```

Comment:
this will throw an error if this field doesn't exist in the `settings.json` file. The example scripts only create the camera settings in the `data["Vehicles"]["Drone1"]["Cameras"]["high_res"]["CaptureSettings"]` location rather than the CameraDefaults.

Reply:
I thought it was in the default settings that everyone would have, but it's probably a good idea to have a default we can fill in if it is missing from the json. I'll work that out :+1: